src/frontends/paddle/tests/paddle_pip_check.py

```diff
@@ -1,14 +1,56 @@
 import pkg_resources
 import re
 import sys
+import os
 
 req_file=sys.argv[1]
 
+constraints = {}
+constraints_path = None
+requirements = []
+
+# read requirements and find constraints file
+with open(req_file) as f:
+    raw_requirements = f.readlines()
+for line in raw_requirements:
+    if line.startswith("-c"):
+        constraints_path = os.path.join(os.path.dirname(req_file), line.split(' ')[1][:-1])
```

Comment:
It's better to create a list to store the 'constraints_path' in case there is multiple '-c' in requirements in the future